contents/handbook/small-teams/marketing/objectives.mdx

```diff
@@ -1,18 +1,36 @@
+      1. being an authority and vital resource on building successful products.
+      2. creating a world-class community experience.
 
-* **Key Results:**
-    * Increase [overall weekly website traffic](https://app.posthog.com/insights/hg6kfoIG) to 27k unique visitors per week (filtering out viral spikes).
-    * Increase [‘first user in org’ signups](https://app.posthog.com/insights/EwWz7QuV) to 600 per week without decreasing the percentage that are ICPs (20-25%).
-    * Make Tutorials our [3rd most visited section](https://app.posthog.com/insights/n8b9SHgd) (after Docs and Blogs)
-    * Increase number of [subscribers](https://newsletter.posthog.com/publish/subscribers) to Product for Engineers to 1,500
-    * Get [200 orgs](https://docs.google.com/spreadsheets/d/1ryDGzXcG0OSH-GoX-zlI2CKwNu-6Zivi7EM32WLdids/edit#gid=0) to sign up to the startup deal
-    * Increase [28 day YouTube watch time](https://studio.youtube.com/channel/UCn4mJ4kK5KVSvozJre645LA?c=UCn4mJ4kK5KVSvozJre645LA) to 227 hours
+* **Goals:**
+    * Nail Twitter and content distribution (**Owner:** TBD, maybe Charles?)
+    * Nail internal content discovery and engagement (**Owner:** Ian with Website & Docs)
+    * Make PostHog for Startups the default choice for technical founders (**Owner:** Joe)
+    * Scale the newsletter to thousands of engaged subscribers (**Owner:** Andy)
+    * Create and execute a community plan (**Owner:** Lior)
+    * Ship printed PMF guide with a distribution plan (**Owner:** Lottie)
 
 * **Rationale:**
-    * We frequently get brand mentions where people are saying ‘Amplitude, Mixpanel or PostHog’. We want them to also say things like ‘LaunchDarkly or PostHog’, ‘FullStory or PostHog’, and 'Optimizely or PostHog'.
-    * While SEO remains very important to us, we want to diversify our audience slightly so we are not so dependent on it as a single channel as a defensive move, especially if there are AI-related changes to search in future. Building things like our newsletter, YouTube channel, and our startup community will help us do this. 
-    * While tutorials often do not drive top of funnel awareness for people who have never heard of PostHog, they will help drive awareness of _other products_ once users are signed up to PostHog. 
+    * Previously we've set metric-centric goals. This made sense when BAU work drove growth easily, but we feel marketing and content needs to become more mission driven to drive long-term bottom-up growth. We still monitor several metrics, but we treat them as indicators of progress rather than discrete goals.
 
-* **Anti-goals** 
-    * Trying a new channel. Working on community. Working with developer influencers. TikTok. 
+    * Our gut instinct is that, while community is a very hard problem to solve, getting it right might be the thing that unlocks our next surge in growth. An engaged and happpy community is a stimulant for word-of-mouth growth. The longer we wait, the harder it'll be to make it work.
```

Comment:
I think figuring out social is probably the thing that unlocks our next surge of growth. For example, this [tweet](https://twitter.com/dannypostmaa/status/1670412104875282433?s=20) drove a massive amount of signups. Many competitors and similar "analytics" companies are entirely focused on driving growth through social.

Community is a much larger hill to climb. It is very likely that if we want to do it, we will have to nuke our support community and start again. No one comes there for non-support or PostHog related reasons which a successful community would require.

Reply:
To your point, I'm kind of wondering if we're just thinking about "community" in the wrong way – i.e. our language is around building, planning, etc., but it's more like a garden you water. To that end, our goals should be (and kind of already are) things that will help that grow – e.g. Nailing Twitter, which is very much a community, startups program (a potential community), newsletter (another community branch), and so on.

Reply:
> Nailing Twitter, which is very much a community, startups program (a potential community), newsletter (another community branch), and so on.

A healthy, non-support driven community is much more likely to come out of one of these rather than the Slack support community we have now.